src/libs/string.liq

```diff
@@ -381,3 +381,21 @@ let string.char.ascii.number = list.init(10, fun (c) -> c + 48)
 def string.char.ascii.random(range=[...string.char.ascii]) =
   string.char(list.nth(range, random.int(min=0, max=list.length(range) - 1)))
 end
+
+# Escape HTML entities.
+# @category String
+def string.escape.html(s) =
+  string.replace(
+    pattern = "[&<>\"']",
+    fun (c) -> begin
+      if c == "&" then "&amp;"
+      elsif c == "<" then "&lt;"
+      elsif c == ">" then "&gt;"
+      elsif c == "\"" then "&quot;"
```

Comment:
```suggestion
      elsif c == '"' then "&quot;"
```
I think it's OK to use single quotes to prevent escaping, but the linter may not be happy.